lib/domain/entity/pokemon/pokemon.dart

```diff
@@ -0,0 +1,13 @@
+class Pokemon {
+  final int id, height, weight;
+  final String name, url;
+  final DateTime dateTime;
+
+  Pokemon(
+      {required this.id,
+      required this.height,
+      required this.name,
+      required this.url,
+      required this.weight,
+      required this.dateTime});
```

Comment:
If you add a comma at the end this will automatically be formatted correctly.
```suggestion
      required this.dateTime,
      });
```
@bhaveshpawar07

Reply:
changes have been applied and pushed to presentation branch